README.md

```diff
@@ -1,31 +1,44 @@
-# Fairing
+# Kubeflow Fairing
 
-Easily train ML models on Kubernetes, directly from your python code, or a Jupyter notebook.
+Easily package, deploy, and run your machine learning (ML) training jobs on Kubernetes,
+directly from Python code or a Jupyter notebook. 
+
+## Features
+
+Kubeflow Fairing provides functions to seemlessly train models remotely on Kubernetes
+clusters.
+
+- Build images in seconds, without Docker
```

Comment:
I'm not sure what the highlighted feature set should be but maybe something like

* Easy API for data scientists to use, but extensible for advanced users
* Run the same workflow locally on your laptop, on a notebook, or in an automated pipeline
* Pluggable preprocessor, builder, and deployer architecture to interface with different environments

Reply:
How about this:

Kubeflow Fairing simplifies the process of packaging, deploying, and running your ML training jobs remotely.

- Easily package your ML training job's Python code and Jupyter notebooks as container images.
- Deploy and run your ML training job on a variety of environments, including: Kubernetes clusters, Kubeflow, or Google Cloud ML Engine.
- Kubeflow Fairing's extensible preprocessor, builder, and deployer architecture provides a way for it to interface with different ML training environments.